.clang-format

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 Intel Corporation.
+# Copyright(c) 2022-2023 Intel Corporation.
```

Comment:
should we use 2022 or 2022-2023? Any guide we're refering to?

Reply:
refering to this page: https://wiki.ith.intel.com/display/ESIF/Updating+Copyright+Notices